### PR TITLE
Utilize addwiki API framework, replacing labs db queries with API queries

### PR DESCRIPTION
**Pull request by MusikAnimal**

### CODE REVIEW

File: src/Controllers/CopyPatrol.php [hunk 1]
```diff
@@ -22,6 +22,8 @@
 namespace Plagiabot\Web\Controllers;
 
 use Wikimedia\Slimapp\Controller;
+use GuzzleHttp;
+use GuzzleHttp\Promise\Promise;
```
Comment (MusikAnimal) [inline src/Controllers/CopyPatrol.php]: Note to self: Don't ever use cURL again. Guzzle it down!

File: src/Dao/EnwikiDao.php [hunk 2]
```diff
@@ -44,26 +48,71 @@ public function __construct(
...
+			}
+		}
+
+		foreach ( $result['pages'] as $page ) {
+			$revisions = $page['revisions'];
+
+			if ( isset( $revisions ) ) {
+				foreach ( $revisions as $revision ) {
+					$data[$revision['revid']] = [
+						'revid' => $revision['revid'],
```
Comment (kaldari) [inline src/Dao/EnwikiDao.php]: It's a bit odd having the revision ID in the data twice. Do we need both of them?

Comment (MusikAnimal) [inline src/Dao/EnwikiDao.php]: Nope. We actually don't need the timestamp either... I can remove both and rename the function to `getRevisionEditors` or something

Comment (kaldari) [inline src/Dao/EnwikiDao.php]: That sounds like a good idea.

**Pull request merged**